resources/js/features/games/hooks/useAllMetaRowElements.ts

```diff
@@ -0,0 +1,279 @@
+import { useMemo } from 'react';
+import { route } from 'ziggy-js';
+
+import { extractAndProcessHubMetadata } from '../utils/extractAndProcessHubMetadata';
+
+export function useAllMetaRowElements(
+  game: App.Platform.Data.Game,
+  allGameHubs: App.Platform.Data.GameSet[],
+) {
+  const developerRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Developer -', 'Hacker -'],
+        primaryLabel: 'Developer',
+        altLabels: ['Hacker'],
+        fallbackValue: game.developer,
+      }),
+    [allGameHubs, game.developer],
+  );
```

Comment:
Hacker isn't being stripped from the raw data:
![image](https://github.com/user-attachments/assets/40565236-504a-44ca-88df-567156ffd050)

![image](https://github.com/user-attachments/assets/0d47483e-dc9a-4030-b099-89426073da2a)

Reply:
Both issues addressed in latest. Note I renamed "RA Features" to simply "RA". Still not 100% sold on that wording; I'm trying to shy away from reusing the word "feature" across multiple rows.

![Screenshot 2025-05-18 at 4 57 36 PM](https://github.com/user-attachments/assets/e2841fcb-ef34-4fd6-8449-24a88c532cb0)

Reply:
Changed to "Achievement Extras" in latest. I couldn't think of anything better.